subsys/net/l2/wifi/wifi_mgmt.c

```diff
@@ -1455,11 +1455,28 @@ BUILD_ASSERT(sizeof(CONFIG_WIFI_CREDENTIALS_STATIC_SSID) != 1,
 	     "CONFIG_WIFI_CREDENTIALS_STATIC_SSID required");
 #endif /* defined(CONFIG_WIFI_CREDENTIALS_STATIC) */
 
+#ifdef CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE
+static const char ca_cert_test[] = {
+#include <wifi_enterprise_test_certs/ca.pem.inc>
+	'\0'};
+
+static const char client_cert_test[] = {
+#include <wifi_enterprise_test_certs/client.pem.inc>
+	'\0'};
+
+static const char client_key_test[] = {
+#include <wifi_enterprise_test_certs/client-key.pem.inc>
+	'\0'};
+#endif /* CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE */
+
```

Comment:
these don't belong in the Wi-Fi management, these have to come from one layer above Wi-Fi management e.g., Wi-Fi shell or Wi-Fi cred of Customer Application. Wi-Fi management just provides `NET_REQUEST_WIFI_ENTERPRISE_CREDS`.

Before upstreaming, these certs reside in Wi-Fi credentials shell.